paper-drawer-panel.html

```diff
@@ -506,7 +505,7 @@
             x <= this.edgeSwipeSensitivity);
       },
 
-      _trackStart: function() {
+      _trackStart: function(event) {
```

Comment:
Great catch! this was the issue behind https://github.com/PolymerElements/paper-drawer-panel/issues/56